packages/nx-plugin/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.29.0",
   "license": "MIT",
   "dependencies": {
-    "@nx/devkit": "^17.1.3",
+    "@nx/devkit": "^17.3.2",
     "tslib": "2.6.2",
-    "nx": "^17.1.3"
+    "nx": "^17.3.2"
   },
```

Comment:
Please use the minimum values here, so that we support older versions as well.

Reply:
Ok, makes sense!